kythe/typescript/indexer.ts

```diff
@@ -1275,9 +1334,11 @@ export function index(
   if (plugins) {
     for (const plugin of plugins) {
       try {
-        plugin.index(
-            (path: string) => getFileVName(path, pathVNames, vname), paths,
-            program, emit);
+        const host = new IndexerHost(vname, pathVNames, paths, program);
```

Comment:
Then down here you can just pass 'visitor' as the host instead.

Reply:
See previous conversation.